forgerock-auth/src/main/java/org/forgerock/android/auth/FRUserKeys.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023 ForgeRock. All rights reserved.
+ *
+ *  This software may be modified and distributed under the terms
+ *  of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth
+
+import android.content.Context
+import org.forgerock.android.auth.callback.DeviceBindingCallback
+import org.forgerock.android.auth.devicebind.UserDeviceKeyService
+import org.forgerock.android.auth.devicebind.UserKey
+import org.forgerock.android.auth.devicebind.UserKeyService
+
+/**
+ * Manage [UserKey] that created by the SDK
+ * The [UserKey] are created with [DeviceBindingCallback]
+ */
+class FRUserKeys(private val context: Context,
+                 private val userKeyService: UserKeyService = UserDeviceKeyService(context)) {
+
+    /**
+     * Load all the [UserKey] which created with [DeviceBindingCallback]
+     * @return All the [UserKey]
+     */
+    fun loadAll(): List<UserKey> {
+        return userKeyService.userKeys
+    }
+
+    /**
+     * Delete the [UserKey]
+     * @param userKey The [UserKey] to be deleted
+     */
+    fun delete(userKey: UserKey) {
+        userKeyService.delete(userKey)
+    }
+
+}
```

Comment:
In the ticket it says "Provide interface to delete all keys.". Are you gonna have a separate method for that or developers should get all the keys and delete one by one?

Reply:
+1 , i like to have deleteAll()

Reply:
I may update the ticket to not delete all keys, in the future, I am thinking of using the same interface to delete keys not only locally but also from the server as well, since user can only delete his/her own key with the session, delete all key make not make sense.

@jeyanthanperiyasamy Yes, has a look at the UserDeviceKeyService, it not only delete the reference, but also the keys, but not sure that is the best class to handle it.

Reply:
this point make sense to me...i didnt think of multiple user can also login in same device. lets keep the same interface